examples/lsm-nice/lsm-nice-common/Cargo.toml

```diff
@@ -8,7 +8,7 @@ default = []
 user = [ "aya" ]
 
 [dependencies]
-aya = { version = "0.11", optional=true }
+aya = { git = "https://github.com/aya-rs/aya", branch = "main", optional = true }
```

Comment:
can we drop branch = "main" from all of these? we should let the repository dictate what its default branch is

Reply:
I thought that this would pick the v0.11.0 tag by default, doesn't look like it.
Will be fixed in the next commit.